velox/exec/WindowFunction.h

```diff
@@ -21,6 +21,13 @@
 
 namespace facebook::velox::exec {
 
+/// Represent different window rank functions.
```

Comment:
This enum can move into the Rank.cpp file now.

Reply:
Oh right, sorry I missed that, removed from here now